app/views/admin/dashboard/index.html.erb

```diff
@@ -52,7 +52,7 @@
     <%= render "govuk_publishing_components/components/list", {
       visible_counters: true,
       items: [
-        link_to("GDS support form (Zendesk)", "https://www.gov.uk/support/internal", class: "govuk-link"),
+        link_to("GDS support form (Zendesk)", "https://support.publishing.service.gov.uk", class: "govuk-link"),
```

Comment:
Using [support_url method](https://github.com/alphagov/whitehall/blob/6c01a6f7262ad42b2a3b574c03dedb44ffbc6fbc/lib/whitehall.rb#L27-L29) will ensure correct link in each environment.
```suggestion
        link_to("GDS support form (Zendesk)", Whitehall.support_url, class: "govuk-link"),
```